packages/treasury/src/types.js

```diff
@@ -183,3 +196,22 @@
  * @param {RelativeTime} recordingPeriod
  * @returns {CalculatorKit}
  */
+
+/**
+ * @typedef {Object} FeeParamManager
+ * @property {() => Record<Keyword,ParamDescription>} getParams
+ * @property {(bigint) => void} updateProtocolFee
+ * @property {(bigint) => void} updatePoolFee
```

Comment:
Without a parameter name, TypeScript parses this as a parameter named `bigint` with type `any` :-/

```suggestion
 * @property {(fee: bigint) => void} updateProtocolFee
 * @property {(fee: bigint) => void} updatePoolFee
```

Reply:
Fixed.  Found other places in this file.  Added a clause to my personal pre-submit. 

Looked in a few other places.